packages/loader/README.md

```diff
@@ -0,0 +1,47 @@
+# Loaders
+
+This package provides a number of React loading elements that you can use to show
+that something on your page is... loading.
+
+## Installation
+
+```node
+yarn add @onaio/loaders
```

Comment:
The README should be about `@opensrp/loader` and not `@onaio/loaders`. I think you should also mention the default props for the default props (configurations) used by OpenERP cc @p-netm @KipSigei